test/main.test.js

```diff
@@ -91,6 +92,37 @@ describe('GotScraping', () => {
         expect(response.statusCode).toBe(200);
     });
 
+    test('should post json', async () => {
+        const body = { foo: 'bar' };
+
+        const response = await gotScraping({
+            responseType: 'json',
+            url: `http://localhost:${port}/jsonPost`,
+            json: body,
+            method: 'POST',
+        });
+
+        expect(response.statusCode).toBe(200);
+        expect(response.body).toMatchObject(body);
```

Comment:
I think `.toMatchObject` allows partial matches. `.toEqual` would be better here.